llvm/docs/Passes.rst

```diff
@@ -199,8 +202,9 @@ information query.
 This pass decodes the debug info metadata in a module and prints in a
```

Comment:
`print<module-debuginfo>` is the name of the pass, there's no `module-debuginfo` analysis, can you update this section?

Reply:
I've changed the name of the pass to `print<module-debuginfo>`. But it is still listed among the analysis passes.
I got a feeling that this document already is a bit messy as it lists analysis passes and transform passes. But many of these print passes that are listed as analysis passes are actually function/module passes. So it would take a bit more work if we want to be exact in this document about how passes actually are classified (but maybe that cassification isn't very important for this document).